xml/System.Activities.Presentation.PropertyEditing/EditModeSwitchButton.xml

```diff
@@ -100,7 +100,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a value that, when <see langword="true" />, means that the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> will be calculated automatically to match the <see cref="P:System.Activities.Presentation.PropertyEditing.PropertyContainer.ActiveEditMode" /> of the owning <see cref="T:System.Activities.Presentation.PropertyEditing.PropertyContainerEditMode" />. Otherwise, the mode to switch to will be based on the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> property.</summary>
+        <summary>Gets a value that, when <see langword="true" />, means that the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> will be calculated automatically to match the ActiveEditMode of the owning PropertyContainer. Otherwise, the mode to switch to will be based on the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> property.</summary>
         <value>
           <see langword="true" /> if the <see cref="T:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton" /> will switch automatically to the edit mode of its <see cref="T:System.Activities.Presentation.PropertyEditing.PropertyContainerEditMode" />; otherwise, <see langword="false" />.</value>
```

Comment:
nit: the otherwise condition isn't clear. Better, "if false, it sitches to the dit mode defined by the TargetEditMode property.